apps/web/src/client/app.js

```diff
@@ -2,6 +2,7 @@
 //* This will be updated based on a router -> loader flow.
 
 import { initAll } from 'govuk-frontend';
+import './components/accessible-autocomplete';
```

Comment:
Move this to `apps/web/src/client/pages/default.js` please. And I will explain the reason behind it in a dev call.